--- xviewer-plugins-3.4.1/meson.build.orig	2024-11-25 11:21:16.000000000 +0100
+++ xviewer-plugins-3.4.1/meson.build	2025-03-06 21:04:01.694422063 +0100
@@ -21,7 +21,7 @@ po_dir = src_root / 'po'
 metainfo_dir = prefix / datadir / 'metainfo'
 gschemas_dir = prefix / datadir / 'glib-2.0' / 'schemas'
 
-plugin_libdir =  prefix / 'lib' / 'xviewer' / 'plugins'
+plugin_libdir =  prefix / libdir / 'xviewer' / 'plugins'
 plugin_datadir = prefix / datadir / 'xviewer' / 'plugins'
 
 champlain = dependency('champlain-0.12')
